etna/models/mixins.py

```diff
@@ -629,25 +638,152 @@ def get_model(self) -> Any:
         return self._base_model.get_model()
 
 
-class SaveNNMixin(SaveMixin):
-    """Implementation of ``AbstractSaveable`` torch related classes.
+def _load_object(class_name, class_parameters):
```

Comment:
I'm not really sure that this function is really necessary, because we extracted all the logic into the `_load_pl_model`. I'll try to remove this function.